From 01ea000a16734fb846323ecba2f0a5b25ec0f0e4 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <contact@chrisaw.io>
Date: Tue, 29 Sep 2026 12:44:39 +0100
Subject: [PATCH 1/1] disable the avf rkpd integration tests on the gsi

the avf integration tests link against the microdroid test library,
which is not built on the gsi because the virtualization sources are
excluded from the manifest sync.
---
 app/tests/avf/Android.bp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/app/tests/avf/Android.bp b/app/tests/avf/Android.bp
index 6b17dc4..f06edcc 100644
--- a/app/tests/avf/Android.bp
+++ b/app/tests/avf/Android.bp
@@ -4,6 +4,8 @@ package {
 
 android_test {
     name: "AvfRkpdAppIntegrationTests",
+    // the virtualization stack is not built on the gsi
+    enabled: false,
     srcs: ["src/**/*.java"],
     static_libs: [
         "MicrodroidDeviceTestLib",
-- 
2.55.0

